apps/system/lockscreen/js/lockscreen.js

```diff
@@ -754,6 +777,12 @@
     }
   };
 
+  LockScreen.prototype.forcePassCodeUnlock =
```

Comment:
As I said in Bug 1186100, this should be renamed and changed for a new implementation.